src/options/options.css

```diff
@@ -20,74 +19,69 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
```

Comment:
Where did you take this numbers from? Are these somewhat official?

Because if I had to choose a margin/padding for that here, I'd try to align the options with how the other Firefox elements are inset at the top of it?
Is this possible in some way? If so, it may look quite nice as it aligns better into the general UI then…

![align](https://user-images.githubusercontent.com/11966684/55160677-29b7c680-5164-11e9-9edb-843ac481bde1.png)


(And BTW, I predict on Android this can cause issues :wink:)

Reply:
![screenshot-1553834447_667x821](https://user-images.githubusercontent.com/1430854/55209943-f20e5480-521f-11e9-9687-cd08cfb43aff.png)

this is how it looks without margin, does it looks a little squeezy?

Reply:
Do _you_ think so?

Reply:
the sides at least

Reply:
Okay fine. Go with the way you like it best… (maybe even a compromise, i.e. not the "original" 32px or what it was)